src/avram/tasks/db/schema_dump.cr

```diff
@@ -4,6 +4,19 @@ class Db::Schema::Dump < LuckyCli::Task
   def initialize(@dump_to : String? = nil, @quiet : Bool = false)
   end
 
+  def help_message
+    <<-TEXT
+    Exports/dumps the database schema to a sql file.
```

Comment:
I like this, but I wonder if it should be noted that this doesn't include the data in the tables? Also, should that be an option? 🤔

Reply:
Good point. And I don't think so on exporting data